src/UI/Component/MainControls/Slate/Prompt.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/* Copyright (c) 2018 Nils Haagen <nils.haagen@concepts-and-training.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\MainControls\Slate;
+
+/**
+ * Prompts are notifications from the system to the user.
+ */
+interface Prompt extends Slate
+{
+	public function	withUpdatedStatusCounter(int $count): Prompt;
```

Comment:
Slates are triggered by bulky buttons, prompts are not, why is this?
Since I can pass counters to glyphs and glyphs to bulky buttons and use these as triggeres I would have expected a very different interface here. But I may miss the point with the Prompts completely.

Reply:
We decided to introduce a Slate and a specialized slate (the Prompt) that's supposed to be those thingys that somehow "alarm" users visually with the counters. We decided to make this counter an issue of the prompt (and not the glyph) because we expect that we will need one type of prompt ("Notifications") that bundles other prompts ("Mail", "Background Task", ...) and want to add counters on this level. So the prompt uses the counter on the glyph to visualize its own counter, but owns the counter conceptually.